BUILD.md

```diff
@@ -142,141 +121,91 @@ At this point, you can build the solution from the command line or open the
 generated solution with Visual Studio.
 
 #### Build the Solution From the Command Line
-
-While still in the build directory:
-
+To build the Debug configuration, run the following command from the build
+directory:
 ```
 cmake --build .
 ```
 
-to build the Debug configuration (the default), or:
-
+To build the Release configuration, run the following command from the build
+directory:
 ```
 cmake --build . --config Release
 ```
 
-to make a Release build.
-
 #### Build the Solution With Visual Studio
-
-Launch Visual Studio and open the "GFXReconstruct.sln" solution file
-in the build folder. You may select "Debug" or "Release" from the Solution
+Launch Visual Studio and open the `GFXReconstruct.sln` solution file
+from the build folder. You may select "Debug" or "Release" from the Solution
 Configurations drop-down list. Start a build by selecting the Build->Build
 Solution menu item.
 
-
-## Building On Linux
-
-### Linux Build Requirements
-
-This repository has been built and tested on the two most recent Ubuntu LTS
-versions. Currently, the oldest supported version is Ubuntu 14.04, meaning
-that the minimum supported compiler versions are GCC 4.8.2 and Clang 3.4,
-although earlier versions may work. It should be straightforward to adapt this
-repository to other Linux distributions.
-
+## Building for Linux
 #### Required Package List
-
 Building on Linux requires the installation of the following packages:
-
-* A C++ compiler
-* GIT
+* A C++ compiler with C++-14 support
+* Git
 * CMake
-* X11, XCB and Wayland development libraries
-
-Additional recommended tools packages include:
-
-* Python 3
+* X11 + XCB and/or Wayland development libraries
```

Comment:
we no longer require Python 3?

Reply:
This project does not require Python to build. It does require Python for code generation, which is an offline process that is not yet documented.